third_party_license_file_generator/site_packages.py

```diff
@@ -223,6 +223,9 @@ def _read_all_module_metadatas_and_license_files(self):
                 continue
 
             if not thing.endswith('dist-info'):
+                if thing.endswith('.egg') and os.path.isdir(os.path.join(path_to_thing, 'EGG-INFO')):
+                    path_to_thing = os.path.join(path_to_thing, 'EGG-INFO')
+                else:
                 continue
```

Comment:
```suggestion
                	continue
```

Syntax error here- Tabbing in `continue` will solve it.